amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -230,15 +230,15 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
                   }
                 </div>
                 <div className="section-title title-3">Created</div>
-                <div>
+                <div className="body-2 text-primary">
                   { formatDateTimeShort({ epochTimestamp: dashboard.created_timestamp}) }
                 </div>
                 <div className="section-title title-3">Last Updated</div>
-                <div>
+                <div className="body-2 text-primary">
                   { formatDateTimeShort({ epochTimestamp: dashboard.updated_timestamp }) }
                 </div>
                 <div className="section-title title-3">Recent View Count</div>
-                <div>
+                <div className="body-2 text-primary">
                   { dashboard.recent_view_count }
                 </div>
```

Comment:
Could you try to use semantic markup here? Maybe it works out of the box.

In this case, we would be something like:
```
<h3 className="section-title title-3">Created</h3>
<p className="body-2 text-primary">{ formatDateTimeShort....}</p>
...
```

Reply:
Here for more about the rationale: https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0

Reply:
Thanks Marcos, will try semantic markup.